.github/workflows/release_sda-admin.yaml

```diff
@@ -4,7 +4,7 @@ on:
     merge_group:
     pull_request:
       paths:
-        - 'sda-admin/.version'
+        - "sda-admin/.version"
```

Comment:
But for the string `sda-admin/.version`, there shouldn't be any difference with the single quote and double quote regarding path parsing, right?